app/src/main/java/com/example/atlaskotlindemo/ui/notifications/NotificationsFragment.kt

```diff
@@ -40,38 +40,15 @@ class NotificationsFragment : Fragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-        (requireActivity().application as AtlasDemoApplication).atlasSdk.bindAtlasView(lifecycle, binding.atlasView)
-        binding.atlasView.openPage()
 
-        binding.atlasView.setAtlasMessageHandler(
-            object : AtlasMessageHandler() {
-                override fun onError(message: String?) {
-                    if (lifecycle.currentState.isAtLeast(Lifecycle.State.RESUMED)) {
-                        Log.d("AtlasView", "onError: $message")
-                    }
-                }
+        // Step 1: Create AtlasViewFragment
+        val atlasSdk = (requireActivity().application as AtlasDemoApplication).atlasSdk
+        val atlasViewFragment = atlasSdk.getAtlasViewFragment()
 
-                override fun onNewTicket(ticketId: String?) {
-                    if (lifecycle.currentState.isAtLeast(Lifecycle.State.RESUMED)) {
-                        Log.d("AtlasView", "onNewTicket: $ticketId")
-                        // (requireActivity().application as AtlasDemoApplication).atlasSdk.updateCustomFields(ticketId, mapOf("customField" to "customValue")
-                    }
-                }
-
-                override fun onChangeIdentity(
-                    atlasId: String?,
-                    userId: String?,
-                    userHash: String?
-                ) {
-                    if (lifecycle.currentState.isAtLeast(Lifecycle.State.RESUMED)) {
-                        Log.d(
-                            "AtlasView",
-                            "onChangeIdentity: $atlasId $userId $userHash"
-                        )
-                    }
-                }
-            }
-        )
+        // Step 2: Replace the current fragment with AtlasViewFragment in full screen
+        childFragmentManager.beginTransaction()
+            .replace(binding.fragmentContainer.id, atlasViewFragment)
```

Comment:
Just for a clarity, can we name it `binding.atlasFragmentPlaceholder`? Or is **Container** is a common prefix?

Reply:
How about `atlasFragmentContainer `

Reply:
`binding.fragmentContainerView`